### PR TITLE
Sneak peek: Math magicians

### PR DESCRIPTION
**Pull request by armandocomellas1**
### The main purpose of this pull request are:

- Setup react project
- Understand how to use medium-fidelity wireframes to create a UI.

💯

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-# Math Magicians: setup
+# Getting Started with Create React App
 
-This project is about creating a math calculator with 3 pages but this time builded with react.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Built With
+## Available Scripts
```
Comment (ShahierNashaat) [inline README.md]: - [ ] Please make sure that the readme file is like the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md).

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @armandocomellas1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (armandocomellas1) [reply README.md]: CHanges are done, thank you

**Review by smunozmo — APPROVED** [review]
Hi @armandocomellas1, 👋 

## Highlights 👏 

- All the suggested changes are made, excellent job! 😎 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**